### PR TITLE
Syndicate Cloak

### PR DESCRIPTION
**Pull request by MisterVAdos**
## About the PR
added: Syndicate Cloak 

## Media

Cloak of the Syndicate.
Adds a new Syndicate Cloak stylized to match their style.

image
image
image
image
image

## Breaking changes

**Changelog**
:cl:
- add: Added the syndicate cloak.

### CODE REVIEW

File: Resources/Textures/Clothing/Neck/Cloaks/Synd.rsi/Prototype.txt
Comment (Errant-4) [inline Resources/Textures/Clothing/Neck/Cloaks/Synd.rsi/Prototype.txt:10]: Do not put an entity prototype in Resources/Textures, and do not put yaml code in a .txt file

This should go somewhere in /home/useriel/s14/Resources/Prototypes/Entities/Clothing/Neck/
Probably cloaks.yml

Comment (Errant-4) [reply Resources/Textures/Clothing/Neck/Cloaks/Synd.rsi/Prototype.txt:10]: Should this be a Steal Target if its not spawning anywhere on the station?

**Comment by TheShuEd** [comment]
Closed due to feature freeze May 10th-June 14th. Comment to have it reopen after this.

**Comment by Ghagliiarghii** [comment]
I love this texture, please reopen after freeze :+1:

File: Resources/Textures/Clothing/Neck/Cloaks/Synd.rsi/Prototype.txt
Comment (MisterVAdos) [reply Resources/Textures/Clothing/Neck/Cloaks/Synd.rsi/Prototype.txt:10]: Sorry. I have only recently started to understand this matter, and so far I lack experience...

**Comment by MisterVAdos** [comment]
Good morning, can you please defrost mine "PR" ?

**Review by AJCM-git — CHANGES_REQUESTED** [review]
this needs to be obtainable in-game

**Comment by Emisse** [comment]
are you still working on this

**Comment by Plykiya** [comment]
> are you still working on this

if the only blocker is making it obtainable, I can make a followup PR to do that.

File: Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml
Comment (UbaserB) [inline Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml:279]: This description makes no sense and the first letter should be capitalised so in this case it would be "Dark"

File: Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml
Comment (UbaserB) [inline Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml:277]: It should be Syndicate not Synd

File: Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml
Comment (UbaserB) [inline Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml:282]: Syndicate, not Synd again

**Comment by UbaserB** [comment]
> this needs to be obtainable in-game

to add on to this, you can add it to the nukie shuttle where the current mantle thing (iirc) is right now

**Comment by slarticodefast** [comment]
Hey, we had a vote among the maintainers and the conclusion was that the sprite needs some adjustments before it can be merged. Here the feedback from one of our art directors:
- the detail in the shading on the back of the cloak is too intense and detracts away from the syndicate logo which doesn't have enough contrast
- the cloak should probably be generally darker as well
- the issues are mostly with the back worn and icon sprite, the front and sides are roughly good enough

Since you did not answer emisse if you plan to continue working on this, I will close this PR for now. Leave a comment when you come back to this, and I will re-open it. Thank you for your contribution.

**Pull request closed without merging**
